.travis.yml

```diff
@@ -28,9 +28,23 @@ before_script:
 
 script:
   - make test-travis
+  - make
+  - make release
+
+deploy:
```

Comment:
It wont erase the release made by Goreleaser?

Reply:
I assumed it would all be aggregated under the same release, but I should check first

Reply:
Maybe checkout https://goreleaser.com/ci/
This could simplify the `Makefile` within goreleaser step.